### ISSUE TITLE
When adjusting tabs to Icons reverts on reboot.

### ISSUE THREAD

**Issue by evanquinnalaska** [Birkbjo/OctoPrint-Themeify#32]
I installed the latest version of Octoprint 15.1 plus the latest of Themeify last night and noticed that one of the tabs reverts back to its name after rebooting Octoprint. 

I am using fa fa server and the tab is OctoPrint-Enclosure.

To fix it I simply flip the dongle next to the Line Item from ON > Off > ON > Save and then refresh the page and its back to being a Icon vs the name.  Reboot and its back. 

Not sure if this is a problem with OctoPrint-Enclosure or Themeify.

**Comment by SillyHats** [comment]
I am having the same issue with the latest stable release versions as well.

**Comment by Birkbjo** [comment]
Thanks for the reports, sorry for the delayed response and resolve time on these issues. I've been having a lot of other stuff to do, and will have some time soon to resolve the open issues. 
Do you get the same error log in the console as #35 describes?

**Comment by evanquinnalaska** [comment]
I'll take a look when I get home tonight.

Evan Quinn
Phone #: 1-907-299-9009

**Comment by OutsourcedGuru** [comment]
I don't think that it's failing at the fifth ordinal. I've done some _in situ_ testing and it feels like the cutoff is built-in versus custom tabs.
